site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -87,6 +91,7 @@ Flag explanations:
 * num-nodes: The number of nodes to be created in each of the cluster's zones. Default: 4. Depending on the needs of your game, this parameter should be adjusted.
 * no-enable-autoupgrade: Disable automatic upgrades for nodes to reduce the likelihood of in-use games being disrupted.
 * machine-type: The type of machine to use for nodes. Default: n1-standard-4. Depending on the needs of your game, you may wish to [have smaller or larger machines](https://cloud.google.com/compute/docs/machine-types).
+* enable-ip-alias: Use [Alias IP ranges](https://cloud.google.com/vpc/docs/alias-ip) instead of routes based networking. This is required to create windows node pools.
```

Comment:
For this block, since it's a bulleted list, you'll need to put the whole list into

{{% feature publishVersion="1.14.0" %}}
...
{{% /feature %}}

and not just this one line (which is a pain, but how the rendering works).